Neos.Flow/Classes/Http/Cookie.php

```diff
@@ -142,8 +142,10 @@ public function __construct($name, $value = null, $expires = 0, $maximumAge = nu
         if ($path !== null && preg_match(self::PATTERN_PATH, $path) !== 1) {
             throw new \InvalidArgumentException('The parameter "path" passed to the Cookie constructor must be a valid path as per RFC 6265, Section 4.1.1.', 1345123078);
         }
-        if (!\in_array($sameSite, [self::SAMESITE_LAX, self::SAMESITE_STRICT, self::SAMESITE_NONE, null], true)) {
-            throw new \InvalidArgumentException('The parameter "sameSite" passed to the Cookie constructor must be a valid samesite value. Possible values are "none", "strict", "lax" and null', 1584955500);
+
+        $sameSite = is_null($sameSite) ? self::SAMESITE_LAX : $sameSite;
```

Comment:
But why even allow null? It's optional with the default value now, and `null` doesn't need to be accepted, eh?
```suggestion
        $sameSite = $sameSite === null ? self::SAMESITE_LAX : $sameSite;
```

Reply:
But if we change that the change will be breaking the behavior completely for this case ... now the will just get the new default cookie value but without null transformation the will always end in the exception.

Think that would be bit hard or not?

Reply:
But that would only happen if someone explicitly set it to `false`… because it's either set by the user or coming from the default settings. No?

Reply:
Adjusted the tests now and removed the conversion. But when you have user land code that set sameSite to null in the construct it will break the behaviour with the exception.

But the code change is minimal but need to be done.

Reply:
True. What do the others think? @albe @robertlemke @jonnitto